src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discount20 = 20;
+  const discount50 = 50;
+  let totalAmount = 0;
+
+  totalAmount = price * days;
+
+  if (days >= 7) {
+    totalAmount = totalAmount - discount50;
+  } else if (days >= 3) {
+    totalAmount = totalAmount - discount20;
+  };
+
+  return totalAmount;
```

Comment:
No need to reasign value - you can return if some condition is true. Also do not use else statement.

```suggestion
  const basicCost = price * days;

  if (days >= 7) {
    return basicCost - discount50;
  } 
  if (days >= 3) {
    return basicCost - discount20;
  };

  return basicCost;
```